Classes/Command/QuotaCommand.php

```diff
@@ -81,6 +82,8 @@ protected function execute(InputInterface $input, OutputInterface $output)
                 $this->checkThreshold($storage, $currentUsage);
             }
         }
+
+        return 0;
```

Comment:
Necessary to prevent `Return value of ... must be of the type integer, none returned` in TYPO3 10.